src/lib/useGlobalSpaceDown.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+
+export default (callBack: () => unknown, disabled?: boolean) => {
+	const handleKeyDown = (event: KeyboardEvent) => {
+		if ((event.target as HTMLElement).tagName == 'INPUT' || event.key != ' ') {
```

Comment:
I wonder if this would be better as:

```suggestion
		if ((event.target as HTMLElement).tagName != 'BODY' || event.key != ' ') {
```

That would still allow spacebar selection of select menus, links, etc.